utils/zerovec/README.md

```diff
@@ -38,6 +38,14 @@ Benchmark results on x86_64:
 
 \* *This result is reported for `Vec<String>`. However, Serde also supports deserializing to `Vec<&str>`; this gives 1.8420 μs, much faster than `Vec<String>` but a bit slower than `zerovec`.*
 
+| Operation | `HashMap<K,V>`  | `LiteMap<K,V>` | `ZeroMap<K,V>` |
```

Comment:
Suggestion: To make the table easier to reproduce, please add, somewhere, the name of the benchmark that each cell corresponds to.

Reply:
Done. Added a comment.